src/transformers/trainer.py

```diff
@@ -1139,7 +1167,23 @@ def call_model_init(self, trial=None):
 
         return model
 
+    def ipex_optimize_model(self, model, training=False, dtype=torch.float32):
+        if not training:
```

Comment:
Looks like the `ipex` import can go here.

Reply:
Have made changes accordingly, thanks.